test/test_jit.py

```diff
@@ -9652,21 +9654,6 @@ def func(x):
 #     use_as_constant (should the submodule be listed in __constants__?)
 # )
 nn_module_tests = [
-    ('AlphaDropout', (), ((S,),)),
-    ('Dropout', (), ((S,),)),
-    ('Dropout2d', (), ((S, S),)),
-    ('Dropout3d', (), ((S, S, S),)),
-    ('FeatureAlphaDropout', (), ((S, S),)),
-    ('Hardshrink', (), ((S,),)),
-    ('PReLU', (), ((S,),)),
-    ('PairwiseDistance', (), ((S, S), (S, S))),
-    ('RReLU', (), ((S,),)),
-    ('Sigmoid', (), ((S,),)),
-    ('Softshrink', (), ((S,),)),
-    ('Softsign', (), ((S,),)),
-    ('Tanh', (), ((S,),)),
-    ('Tanhshrink', (), ((S,),)),
-    ('Threshold', (2., 2.), ((S,),)),
     ('Sequential', (torch.nn.Sigmoid(), torch.nn.Threshold(1., 2.)), ((S,),), True),
```

Comment:
why leaving one here?

Reply:
It wasn't in the existing big list of tests. Since it's a big code move I didn't want to add in this PR, we should be able to add a module test for this and remove this list entirely in a small followup.